docs/data-sources/dbaas_redisdb_replica_set.md

```diff
@@ -31,22 +31,22 @@ data "ionoscloud_redis_replicaset" "example" {
 
 * `id` - (Optional) The ID of the RedisDB Replica Set.
 * `display_name` - (Optional) The display name of the RedisDB Replica Set.
-* `location` - (Required) The location of the RedisDB Replica Set.
+* `location` -  The location of the RedisDB Replica Set.
```

Comment:
@avirtopeanu-ionos this should remain `(Required)` since this is an attribute that it is passed to the data source.

Reply:
oops